frontend/components/ExpenseForm.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react';
+import { validateAll } from 'indicative';
+import { Mutation } from 'react-apollo';
+import { Button, Card, Form, Label } from 'semantic-ui-react';
+import Link from 'next/link';
+import InputField from './commons/InputField';
+import formatErrors from '../lib/formatErrors';
+import useFormFields from './hooks/useFormFields';
+import useInputFile from './hooks/useInputFile';
+import { validateFile, NON_NEGATIVE, DATE, required } from '../lib/validation';
+import { EXPENSE_CLAIM } from '../graphql/queries';
+import SuccessMessage from './commons/SuccessMessage';
+import ErrorMessage from './commons/ErrorMessage';
+import InfoMessage from './commons/InfoMessage';
```

Comment:
Personal preference

When importing a lot of items from the same folder in a single file, maybe it would make sense to create and index file for them that re-exports them, so you can import as `import { SuccessMessage, ErrorMessage, InfoMessage } from './commons'` instead. Same for hooks